### PR TITLE
feat(solana) add solana details

### PR DESCRIPTION
**Pull request by PeterBenc**
## Notes

 - start reviewing from [feat(suite): fetch solana conversion rates](https://github.com/trezor/trezor-suite/commit/95cb4491006d3a1fe0e8df550fbce26f073c9ae3), the rest is from other PRs on which this is based
- It works with [this branch](https://github.com/vacuumlabs/trezor-firmware/tree/solana-poc) in the Vacuumlabs’ Trezor firmware fork

## Description

 - add couple of simpler solana details like solana conversion rates, icon, receive modal and network title

### CODE REVIEW

File: packages/components/src/images/coins/sol.svg
Comment (PeterBenc) [inline packages/components/src/images/coins/sol.svg:1]: Let me know what you think about the icon, I tried to follow what you guys have for other icons, e.g. white "letter" inside colorful in colorful circle.

Comment (matejkriz) [reply packages/components/src/images/coins/sol.svg:1]: I've asked our product guys for feedback. I'll let you know.

File: suite-common/icons/assets/cryptoIcons/sol.svg
Comment (matejkriz) [inline suite-common/icons/assets/cryptoIcons/sol.svg:1]: `yarn generate-icons` should be called as described in https://github.com/trezor/trezor-suite/tree/3da85c5dd423243820ebbcc292098c811ece010a/suite-common/icons#how-to-add-or-update-icon to normalize the raw SVG

**Comment by matejkriz** [comment]
Last 4 commits looks good, only the `yarn workspace @suite-common/icons generate-icons` should be applied on `suite-common/icons/assets/cryptoIcons/sol.svg` and https://github.com/trezor/trezor-suite/pull/9256 merged first and conflicts resolved 👌 

But discovery of Solana accounts is working for me on this branch (maybe because of wiser-green-owl.solana-mainnet.discover.quiknode.pro/c218d0c9e451f6c5c0bf8b7c7d8e5a72384c9570/?)

**Comment by PeterBenc** [comment]
@matejkriz I rebased this and ran `generate-icons`. As far as I am concerned this is ready to be merged, any response from the product guys? Anyway, its just an icon so If you guys find a better different one before the release it should super simple to replace this one.

File: packages/components/src/images/coins/sol.svg
Comment (matejkriz) [reply packages/components/src/images/coins/sol.svg:1]: there was some feedback regarding consistency, but no result from it, so let's go with this and we will see if product guys prepare something different later 👌

File: suite-common/icons/src/icons.ts [hunk 1]
```diff
@@ -103,15 +103,14 @@ export const cryptoIcons = {
     name: require('../assets/cryptoIcons/name.svg'),
     nmc: require('../assets/cryptoIcons/nmc.svg'),
     regtest: require('../assets/cryptoIcons/regtest.svg'),
+    sol: require('../assets/cryptoIcons/sol.svg'),
     tada: require('../assets/cryptoIcons/tada.svg'),
     test: require('../assets/cryptoIcons/test.svg'),
-    tsep: require('../assets/cryptoIcons/tsep_tgor.svg'),
-    tgor: require('../assets/cryptoIcons/tsep_tgor.svg'),
+    tsep_tgor: require('../assets/cryptoIcons/tsep_tgor.svg'),
```
Comment (matejkriz) [inline suite-common/icons/src/icons.ts:109]: sorry, there is probably some mistake in `generate-icons` script, but this change is unintentional, I guess. Could you please revert this change? feel free to force push it and I'll merge it

Comment (matejkriz) [reply suite-common/icons/src/icons.ts:109]: nevermind, it has been fixed in https://github.com/trezor/trezor-suite/pull/9239

**Review by matejkriz — APPROVED** [review]

**Pull request merged**